Portfolio/index.html

```diff
@@ -7,7 +7,34 @@
   </head>
 
   <body style="margin: 0; padding: 0">
-    <!-- Write your code below for PORTFOLIO -->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+    <h3>Deo</h3>
+    <ul>
+            <li>ABOUT</li>
+            <li>SKILLS</li>
+            <li>REVIEWS</li>
+            <li>PROJECT</li>            
+     </ul>
+     <img src="css.icon.png"><br>
+     <button><a href="https://mail.google.com/mail/u/1/?hl=en#inbox?compose=new">Contact</a></button>
+     <h1><bold>Jhon Doe</bold></h1>
+     <img src="../Portfolio/public/jhonDoe.png" height="100px" width="100px">
+     <h1><bold>Front End Developer</bold></h1>
+     <h3>GET TO KNOW ME</h3>
```

Comment:
The 'src' attribute for the image tag is missing. Provide the correct path to the JavaScript icon image file.